docs/usage-guide.md

```diff
@@ -769,22 +768,26 @@ Supported authentication methods:
 A good security practice is to store the password securely. In the example below we use bcrypt (currently the only supported hash in Chroma server side auth) to hash the plaintext password.
 :::
 
+To generate the password hash, run the following command. Note that you will need to have `htpasswd` installed on your system.
```

Comment:
Can we tell you how to install htpasswd. The reason we used docker here is because it was an assumed dependency

Reply:
Discussed: users configuring auth should know enough to be able to install system utilities on their platform.